### PR TITLE
Add azure-nextgen-py-static-website example

### PR DESCRIPTION
**Pull request by lblackstone**

### CODE REVIEW

File: azure-nextgen-py-static-website/Pulumi.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: azure-nextgen-py-static-website
+runtime:
+  name: python
+  options:
+    virtualenv: venv
+description: An example of a static website hosted on Azure Blob Storage + Azure CDN.
+template:
```
Comment (mikhailshilkov) [inline azure-nextgen-py-static-website/Pulumi.yaml:7]: I guess you copied this from the other example. But do you know if this actually makes sense here? I thought `template` would only apply to `pulumi new`?

File: azure-nextgen-py-static-website/__main__.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import pulumi
```
Comment (mikhailshilkov) [inline azure-nextgen-py-static-website/__main__.py:1]: Copyright header?

File: azure-nextgen-py-static-website/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pulumi>=2.0.0,<3.0.0
+pulumi-random>=3.0.0
+pulumi-azure-nextgen>=0.1.0
```
Comment (mikhailshilkov) [inline azure-nextgen-py-static-website/requirements.txt:3]: Let's require 0.6.1 - that's where automatic location got introduced

File: azure-nextgen-py-static-website/__main__.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+import pulumi_random as random
+
+# TODO: Remove after autonaming support is added.
+random_suffix = random.RandomString("randomSuffix",
+    length=10,
+    special=False,
+    upper=False)
+
+config = pulumi.Config()
+location = config.get("location")
```
Comment (mikhailshilkov) [inline azure-nextgen-py-static-website/__main__.py:12]: 0.6.1 has auto-location population. So, we don't need this config: instead, we should suggest using `pulumi config set azure-nextgen:location westus`.

**Review by mikhailshilkov — APPROVED** [review]
LGTM, just a few small remarks

File: azure-nextgen-py-static-website/Pulumi.yaml (hunk 1 above)
Comment (lblackstone) [reply azure-nextgen-py-static-website/Pulumi.yaml:7]: Will remove.

**Pull request merged**